conda_build/cli/main_build.py

```diff
@@ -231,7 +237,7 @@ def execute(args):
     else:
         api.build(args.recipe, post=args.post, build_only=args.build_only,
                    notest=args.notest, keep_old_work=args.keep_old_work,
-                   already_built=None, config=config)
+                   already_built=None, config=config, noverify=args.verify)
```

Comment:
OK, you flipped both of these, so I have my same complaint as before.  The logic flip should only happen in one place - ideally in the argument.  To me, that means that you should always have the keyword and the argument agree (verify=args.verify or noverify=args.no_verify)  Pick one.